### ISSUE TITLE
Post install setup

### ISSUE THREAD

**Issue by michaelriordan** [directus/docs#59]
https://docs.directus.io/install.html says on first run users will be guided through the setup process at /admin. In fact, a login screen appears. Have not yet been able to figure out exactly how to run a manual installation yet as your documentation is so poor...

**Comment by benhaynes** [comment]
Hello @michaelriordan – there are _many_ ways to install Directus. Individually, from source, cloning, zip, CLI, Docker, etc... can you be a bit more specific about the setup you're having issues with?

If you're seeing the login page then your API is likely already installed. For the Combined build, `/admin` is both the login and install page... it shows different messages depending on if you have a default API configured or not.

Hope that helps... but please share more info on your setup when you have a chance. Thanks!

**Comment by rijkvanzanten** [comment]
If you point the app to an API that doesn’t have a config file, the app will automatically start the installation flow. As @benhaynes mentioned, there are lots of ways how you could’ve installed the platform, so it’s a bit hard to pinpoint why it’s not showing for you exactly.

**Comment by BenjaminF1999** [comment]
Unfortunately I ran into the same issue.
I followed the instructions on (https://docs.directus.io/install.html#configure) as follows:
1) I created a subdomain for my website with the document root set to: /public
2) Navigated to /admin within my browser
Just as @michaelriordan mentioned a login screen pops up with the error "Couldn't reach API"
The next step on the instruciton page is: Follow the prompts to complete configuration (see steps below)
I'm not sure what to do next, because there are no prompts at all. I tried a manual installation, but ran into other problems...
I really hope anyone can help

**Comment by rijkvanzanten** [comment]
@BenjaminF1999 can you provide some specifics on the server setup you're using? With all the variables in server setup, it's hard to give you a definitive answer on what's wrong / how to proceed 🙂 

(Also please verify if your php user (likely `www-data`) has write access to the directus folder, if you have `mod_rewrite` enabled (if using Apache), and if you have `php_curl` installed).

**Comment by rijkvanzanten** [comment]
> as your documentation is so poor...

the docs are, just like the rest of the project, a community effort. Feel free to propose changes to the docs here: https://github.com/directus/docs @michaelriordan

**Comment by BenjaminF1999** [comment]
Hey @rijkvanzanten ,
thank you so much for replying so fast! I appreciate it a lot.
Iam running Debian 9 on a VPS with Apache2 installed. The API seems to work when going to my directus domain, obviously it shows the error 

> This Directus API instance has not been configured. Install via the Directus App (eg: /admin) or read more about configuration at: https://docs.directus.io/install.html#configure

But when I try to go to /admin it shows the login screen of directus instead of a configuration screen

**Comment by rijkvanzanten** [comment]
Could you check the dev tools of the app? I suspect that it might not got the api response it was looking for. Could you check if the CORS headers are in the api

**Comment by romaiiinnn** [comment]
Hello guys,
I just tried installing Directus 7 Suite after a lot of reading on your amazing CMS 👍 

But as described by others here, I ran into troubles while trying to install it on my mac for local tests. I tried the automatic installation of the Suite : 
git clone https://github.com/directus/directus.git

"1- Set your document root to the /public directory"

What I did here to run it locally (I'm on Mac) : 
cd directus/public
php -S localhost:8080

"2- Navigate your browser to the App at /admin"

Here I'm getting a blank page. The network console seems to show that it couldn't load all the assets files (js, css, images). 

Then I tried to run the local server deeper from /admin : 
cd admin/
php -S localhost:8080

And now when going to my localhost:8080 address I'm getting the login screen.
I can't login. Even using the default user (admin@exemple.com) which I assume could be normal as nothing is set.

"3 - Follow the prompts to complete configuration (see steps below)"

I can't access those mentioned steps.

-----
Then I tried the configure script as described here : 
https://docs.directus.io/api/admin/configure-with-script.html

And that worked perfectly. While looking at my database, it created all the tables and my user.

Then running again my local server from /public folder, I'm getting : 
{"error":{"code":108,"message":"User not authenticated"}}

Then running server from /admin, I'm getting the login screen. But my username and password doesn't work, Im getting an error : 
"Unknown error. Try again later. "

What did I do wrong ? :( 

Thanks for the help!

**Comment by benhaynes** [comment]
Hey @romaiiinnn — I'm not the technical expert here, but...

> "1- Set your document root to the /public directory"

This doesn't mean `cd` to the directory, you would need MAMP or something similar on your Mac and would need to create a virtualhost with the root set to `/public`.

Hope that helps!

**Comment by romaiiinnn** [comment]
Hi @benhaynes, thanks for your feedback! 

Actually I saw that the documentation has been updated since, and I managed to make it work on my local environment. I installed the api and app separately thought, using the build version from git. I find it also easier to understand the concept behind when you're new to it, by using separate installation for api and app.

Anyway great work guys! I will surely dive deeper into all your documentation in the coming weeks, I find Directus and its concept really amazing. Keep the good work!
